en/migrations.rst

```diff
@@ -129,21 +140,25 @@ The following ``Bake`` command would create a migration to add a ``products`` ta
         Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121163249_CreateProducts.php
         Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121163249_CreateProducts.php`
 
-.. note::
+You can also use the ``underscore_form`` as the name for your migrations
+i.e. create_products::
+
+        $ bin/cake bake migration create_products name:string description:text created modified
+    
+        Welcome to CakePHP v3.0.13 Console
+        ---------------------------------------------------------------
+        App : src
+        Path: /home/user/Work/php/cakeblog/src/
+        ---------------------------------------------------------------
+        
+        Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php
+        Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php`
```

Comment:
:+1: Ahh you're right. Before we began this conversation I wanted to demonstrate that you could still see the underscore form, but with the version added note I suppose that is moot.
I'll change it!